cmd/jujud/agent/machine/manifolds.go

```diff
@@ -276,6 +277,10 @@ type ManifoldsConfig struct {
 	// LeaseLog represents the internal lease raft log, used to output lease
 	// changes.
 	LeaseLog io.Writer
+
+	// RaftOpQueue represents a way to apply operations on to the raft
+	// instance from the API.
+	RaftOpQueue *queue.BlockingOpQueue
```

Comment:
I don't think this should be a manifold configuration article.
- It provides an execution characteristic rather than a domain concern.
- It is not in the dependency chain - it doesn't depend on anything common and nothing needs to coordinate on it as a dependency.

Just instantiating it on site in the worker will be simpler.

Reply:
We can't, it has to be passed into the APIServer part.